src/Altinn.Platform/Altinn.Platform.Register/Register/Program.cs

```diff
@@ -1,164 +1,286 @@
 using System;
 using System.IO;
+using System.Reflection;
+using System.Text.Json.Serialization;
 
-using AltinnCore.Authentication.Constants;
+using Altinn.Common.AccessToken;
+using Altinn.Common.AccessToken.Configuration;
+using Altinn.Common.AccessToken.Services;
+using Altinn.Platform.Register.Configuration;
+using Altinn.Platform.Register.Filters;
+using Altinn.Platform.Register.Health;
+using Altinn.Platform.Register.Services.Implementation;
+using Altinn.Platform.Register.Services.Interfaces;
+using Altinn.Platform.Telemetry;
 
-using Microsoft.AspNetCore;
+using AltinnCore.Authentication.JwtCookie;
+
+using Microsoft.ApplicationInsights.Channel;
+using Microsoft.ApplicationInsights.Extensibility;
+using Microsoft.ApplicationInsights.WindowsServer.TelemetryChannel;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Builder;
 using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Http;
 using Microsoft.Azure.KeyVault;
 using Microsoft.Azure.KeyVault.Models;
 using Microsoft.Azure.Services.AppAuthentication;
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.Configuration.AzureKeyVault;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
 using Microsoft.Extensions.Logging;
+using Microsoft.IdentityModel.Logging;
+using Microsoft.IdentityModel.Tokens;
+using Microsoft.OpenApi.Models;
+
+using Swashbuckle.AspNetCore.SwaggerGen;
+
+ILogger logger;
+
+string vaultApplicationInsightsKey = "ApplicationInsights--InstrumentationKey";
+
+string applicationInsightsKey = string.Empty;
+
+var builder = WebApplication.CreateBuilder(args);
+
+ConfigureSetupLogging();
+
+SetConfigurationProviders(builder.Configuration);
+
+ConfigureLogging(builder.Logging);
+
+ConfigureServices(builder.Services, builder.Configuration);
+
+builder.WebHost.UseUrls("http://*:5020");
```

Comment:
The port number and protocol is provided through the ASPNETCORE_URLS environment variable in all environments. Including local dev and docker desktop.